gravityflow/gravityflow
# Add Step for the Pods Gravity Forms Add-On

## Description

This adds a step which integrates with the [Pods Gravity Forms Add-On](https://wordpress.org/plugins/pods-gravity-forms/).

The brightness filter applied to the step type icons has also been reduced as the Pods icon was near invisible when not the selected step.

## Testing instructions
- Install Pods: https://wordpress.org/plugins/pods/
- Install Pods GF: https://wordpress.org/plugins/pods-gravity-forms/
- Create a new post type Pod by going to /wp-admin/admin.php?page=pods-add-new
- Create a form to caputre the post title and content
- Create a Pods feed
- Create a workflow with an Approval step and a Pods step
- Submit form
- Confirm Pods feed is not processed
- Approve entry
- Confirm Pods feed is processed

## Automated Test Enhancements
N/A
 
## Screenshots
N/A

## Documentation Changes?
The add-on will need adding to the integrations list: https://docs.gravityflow.io/article/30-integration-with-gravity-forms-add-ons

## Checklist:
